src/client.ts

```diff
@@ -194,6 +194,7 @@ export class OpenFeatureClient implements Client {
     // merge global and client contexts
     const mergedContext = {
       ...OpenFeature.getContext(),
+      ...OpenFeature.getTransactionContext(),
```

Comment:
This needs to be in the try/catch now I think. `getTransactionContext()` runs user code.

Reply:
fixed